app/models/miq_server.rb

```diff
@@ -23,6 +24,7 @@ class MiqServer < ApplicationRecord
   belongs_to              :zone
   has_many                :messages,  :as => :handler, :class_name => 'MiqQueue'
   has_many                :miq_events, :as => :target
+  has_many                :miq_workers, :dependent => :destroy
```

Comment:
NOTE this wasn't `:dependent => :destroy` before (it didn't have any `:dependent` action)  Not sure when if ever we `.destroy` an `MiqServer` record but leaving orphaned miq_workers didn't seem like a good idea